java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -279,6 +280,10 @@ public static Object[] transactionTestResult() {
             0L, // zinterstore(key8, new String[] {zSetKey2, key8})
             new Object[] {zSetKey2, "two", 2.0}, // bzpopmax(new String[] { zsetKey2 }, .1)
             2L, // geoadd(geoKey1, Map.of("Palermo", ..., "Catania", ...))
+            new Double[][] {
+                {13.36138933897018433, 38.11555639549629859},
```

Comment:
Does this work? The test uses arbitrary `assertEquals` without precision

Reply:
Local build and tests finished without failing -- [python version](https://github.com/aws/glide-for-redis/pull/1301/files) also didnt have extra comparison logic implemented by the looks of it

Reply:
ok, but you're walking on extremely thin ice

Reply:
Introduced a separate transaction test only containing geo commands, its it justified to do so?

Reply:
In https://github.com/aws/glide-for-redis/pull/1284 I rework transation IT and add a function for geocommands only.
Meanwhile you can simplify your test by using simple coordinates like `10, 20`. Keep in mind that we don't test Redis - we test the client - and we are not obliged to use true real coordinates.

Reply:
using simple values would still require an expected result with high decimal count, since FP accuracy always starts getting introduced starting from the next decimal after the input value's last decimal, for example if we pass in 10.0 we would get: `expected: <10.0> but was: <10.000002086162567>`, and the value `13.36138933897018433` here is what the client comes back with after taking  `13.361389` as input. I have tried this with my local redis instance and the result seems to be stable, so I'm not sure if swapping the value as is makes a difference

I removed my changes adding the new transaction test set after looking at your changes, so for this issue should we keep the more accurate value as our expected test results for now, and wait until aws#1284 gets in and introduce tolerance in the comparisions?